src/components/Navbar.js

```diff
@@ -0,0 +1,103 @@
+            <HStack display={{base: "none",sm: "none", md: "none", lg: "flex"}}>
+                <Image src={logo} style={{width: "40px"}} /> 
+                <Box sx={vr}></Box>
+                <Button variant="ghost"><Link to="dashboard">Начало</Link></Button>
+                <Button variant="ghost">Куестове</Button>
+                <Button variant="ghost">Събития</Button>
+                <Button variant="ghost">Как да стигна?</Button>
+                <Button variant="ghost">Забавни спомени</Button>
+                <Button variant="ghost">Полезно</Button>
+            </HStack>
+
+            <IconButton
+                display={{base: "flex",sm: "flex", md: "flex", lg: "none"}}
+                color="blue.400"
+                aria-label='Options'
+                icon={<HamburgerIcon />}
+                variant='ghost'
+                onClick={onToggle}
+            />
+            
+            <Slide 
+                direction='left' 
+                in={isOpen} 
+                style={{ zIndex: 10 }}
+            >
+                <Flex sx={burger}
+                    display={{sm: "flex", md: "flex", lg: "none"}}
+                >
+                    <Flex flexDir="column">
+                        <Button variant="ghost" mb="5px"><Link to="dashboard">Начало</Link></Button>
+                        <Button variant="ghost" mb="5px">Куестове</Button>
+                        <Button variant="ghost" mb="5px">Събития</Button>
+                        <Button variant="ghost" mb="5px">Как да стигна?</Button>
+                        <Button variant="ghost" mb="5px">Забавни спомени</Button>
+                        <Button variant="ghost" mb="5px">Полезно</Button>
+                        <Button variant="ghost" mb="5px">Настройки</Button>
+                    </Flex>
+                    <Flex justify="flex-end">
+                        <IconButton  
+                            color="blue.400"
+                            aria-label='Options'
+                            icon={<HamburgerIcon />}
+                            variant='ghost'
+                            onClick={onToggle}
+                        />
+                    </Flex>
+                </Flex>
+            </Slide>
+
+            <Heading fontSize="18px" display={{sm: "flex", md: "flex", lg: "none"}}>Начало</Heading>
+
+            <HStack>
+                <AvatarGroup spacing="1em">
+                    <Avatar bg="none"  border="none" icon={<BellIcon boxSize="6"/>}>
+                        <AvatarBadge boxSize="1em" bg="#44818B" borderColor="#44818B">
+                            <Text fontSize="xs" color="white">3</Text>
+                        </AvatarBadge>
+                    </Avatar>
+                    <Avatar size="sm" border="none" name={auth.data.firstName} bg="#44818B"/>
+                    <Text display={{base: "none",sm: "none", md: "none", lg: "flex"}}>{auth.data.firstName} {auth.data.lastName[0]}</Text>
```

Comment:
Please make this a drop-down list with two options: 1) Profile and 2) Logout.

The current Logout logic is implemented in the /profile route, but since you deleted the dummy `<Link to="../profile">Profile</Link>` button, there is no way from the UI to go there and log out.

Reply:
Done, however, I changed the element in Logout.js to be rendered as Text and not Button element.